mac-setup-elevated.sh

```diff
@@ -83,6 +83,9 @@ grep -q github.com ~/.ssh/known_hosts 2>/dev/null || \
     echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" \
         >> ~/.ssh/known_hosts
 
+export # Add homebrew to path on M1 macs
+/opt/homebrew/bin:$PATH
```

Comment:
Um, does this work? I don't really see how it can, but maybe? I've never seen an export statement written like this, so even if it does I'm a little... put off by it?

Reply:
That's part of why this is a draft. In moving from https to ssh github and then trying to move from intellij to vscode, I fat-fingered something. (The patch I moved from the clone done via http worked fine, but this clearly does not.)

Reply:
Doh! I didn't realize this was a draft, sorry!